tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -3,7 +3,7 @@
 <suppressions>
     <!-- These suppressions define which files to be suppressed for which checks. -->
     <suppress files=".+[\\/]internal[\\/].+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
-    <suppress files=".+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck|NullAnnotationsCheck" />
+    <suppress files=".+[\\/]dto[\\/].+|.+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck|NullAnnotationsCheck" />
```

Comment:
You've made it suppress all files (not just java files) that are in a dto package. Perhaps instead of an OR operator just add a new `suppress files` line instead.

Reply:
Changed as requested. 

But is it necessary? Or am I wrong assuming that the javadoc checks only applies to .java files?

Reply:
I'm not sure myself. But it is better to be consistent with the other rules.